packages/kubernetes/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: 1.68.2
```

```suggestion
- version: 1.69.0
```

since it is an enhancement (not a bugfix), I think it should be `1.69.0`

---

I've updated the manifest.yaml and the changelog.yaml to version 1.69.0